api/src/main/java/org/everit/authentication/oauth2/ri/dto/OAuth2ConfigurationDTO.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2011 Everit Kft. (http://www.everit.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.everit.authentication.oauth2.ri.dto;
```

Comment:
Don't need the extra dto package, "org.everit.authentication.oauth2.ri" is fine.